eoscostarica/guias.eoscostarica.io
# feat(docs): Add Eosio core section to spanish version

### GH Issue

Resolves #205  

### Steps to test
1. Go to Núcleo EOSIO
1. Check content


#### CheckList
